README.md

```diff
@@ -55,13 +55,14 @@ processor.call(text)
 :emoji_names         - A list of allowed emoji names. (Array<String>)
 :emoji_url_generator - #call'able object that accepts emoji name as argument and returns emoji image URL. (#call)
                        The original implementation is used when the generator returned a falsey value.
+:hostname            - Used to check whether or not each URL of `href` attributes is external site. (String)
```

Comment:
@yujinakayama In the discussion on the Slack channel, we have decided using "exact match". That is, if the `hostname` is "abc.com" and `href` value of `a` tag is "def.abc.com", the `ExternalLink` adds the `rel` and `target` attributes to the `a` tag. In this case, `hostname` looks good to me, but how do you think about this?